sources/pmwebapi.go

```diff
@@ -137,6 +137,17 @@ func newPcpSource() (PcpSource, error) {
 			log.Fatal("NewRequest: ", err)
 		}
 
+		unit := ""
+		if pcpMetrics.Units != "" {
+			unit = "_" + pcpMetrics.Units
+		}
+		if strings.ToUpper(pcpMetrics.Type) == "COUNTER" {
+			pcpMetrics.Name = pcpMetrics.Name + unit + "_total"
```

Comment:
Can you explain this one a little more? Are you referring to making sure that "COUNTER" is always allcaps or something else? If the former, we can be positive that this will always parse as allcaps since he forces the uppercase value of `pcpMetrics.Type` in the comparison.